test/server-options-test.js

```diff
@@ -625,4 +625,72 @@ describe('SOAP Server with Options', function () {
     });
   });
 
+  it('should return soapenv as envelope key when it is set to soapenv', function (done) {
+    test.server.listen(15099, null, null, function () {
+      test.soapServer = soap.listen(test.server, {
+        path: '/stockquote',
+        services: test.service,
+        xml: test.wsdl,
+        uri: __dirname + '/wsdl/strict/',
+        envelopeKey: 'soapenv'
+      }, test.service, test.wsdl);
+      test.baseUrl = 'http://' + test.server.address().address + ":" + test.server.address().port;
+
+      //windows return 0.0.0.0 as address and that is not
+      //valid to use in a request
+      if (test.server.address().address === '0.0.0.0' || test.server.address().address === '::') {
+        test.baseUrl = 'http://127.0.0.1:' + test.server.address().port;
+      }
+      // console.log(test.baseUrl);
+      axios.post(
+        test.baseUrl + '/stockquote',
+        '<soapenv:Envelope' +
+        ' xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/"' +
+        ' xmlns:soap="http://service.applicationsnet.com/soap/">' +
+        '  <soapenv:Header/>' +
+        '  <soapenv:Body>' +
```

Comment:
Should there not be `</soapenv:Body>` or at least the above be `<soapenv:Body/>`?

Reply:
@w666 would you agree the Body tag above and below are not closed?

Reply:
Yeah, looks like it should be closed
```suggestion
        '  <soapenv:Body/>' +
```